packages/runtime/src/next/components/head-tags.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentPropsWithoutRef } from 'react'
+import { useIsPagesRouter } from '../../components/hooks/useIsPagesRouter'
+import Head from 'next/head'
+
+export function PageTitle({ children, ...props }: ComponentPropsWithoutRef<'title'>) {
```

Comment:
Is there a reason we're explicitly destructuring `children`?

Reply:
Style preference, in our builder we tend to separate children and pass it explicitly. I understand the use cases there are a bit different since we usually wrap the children in other elements, but I prefer the explicitness.